src/tests/intg/test_secrets.py

```diff
@@ -100,11 +100,11 @@ def setup_for_secrets(request):
         [secrets]
         max_secrets = 10
         max_payload_size = 2
+        client_idle_timeout = 10
```

Comment:
@lslebodn: I'm just quoting the commit message as I do believe it answers your question about the 10 seconds: 
"The client timeout in the test has to be at least 10 seconds because
internally, the responders don't allow a shorter timeout."